cmd/ferretdb/ping.go

```diff
@@ -22,29 +22,35 @@ import (
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 	"go.uber.org/zap"
-
-	"github.com/FerretDB/FerretDB/internal/util/ctxutil"
 )
 
-// ping creates connection to FerretDB instance specified by the flags, and runs `ping` command against it.
-// The check is only executed if --setup-database flag is set.
-func ping() {
-	logger := setupLogger(cli.Log.Format, "")
-	checkFlags(logger)
+// ReadyZ represents the Readiness probe, which is used to run `ping`
```

Comment:
I'm still not sure about the whole idea of ping, so please feel free to ignore my comment if it doesn't make sense.

Between having this called "ping" and "readyz", I'd prefer `ping` because it's more universal. Readiness probe is quite a Kubernetes-specific thing, and `ping` looks more general to me. So, I'd continue calling it `ping` :) 

Is there a separate PR that adds the necessary readiness probe logic to the debug handler?

Reply:
Sure, I'll rename it in a moment.

> Is there a separate PR that adds the necessary readiness probe logic to the debug handler?

It's already merged, and being used inside of `debug/debug.go`. The only thing that needs to be done is to provide the actual Probe function to the `debug.Handler` struct which is done in this PR

Reply:
> Sure, I'll rename it in a moment.

#4306 is very clear that documentation from #4424 should be used, and that PR (approved by everyone and merged) is very clear what paths should be used. You updated the code, but did not update the docs. Prevert the path change.

> Between having this called "ping" and "readyz", I'd prefer ping because it's more universal

I'm sorry, but that's just too late.